src/pybind/rgw/rgw_file.pyx

```diff
@@ -406,6 +407,7 @@ cdef class LibRGWFS(object):
                 'f_ffree': statbuf.f_ffree,
                 'f_favail': statbuf.f_favail,
                 'f_fsid': statbuf.f_fsid,
+                'f_fsid2': statbuf.f_fsid2,
```

Comment:
pushed the suggested change to ceph/ceph:wip-11820, so gitbuilders can test the change.